tests/SCsub

```diff
@@ -9,6 +9,8 @@ env_tests = env.Clone()
 # Include GDNative headers.
 if env["module_gdnative_enabled"]:
     env_tests.Append(CPPPATH=["#modules/gdnative/include"])
+    if env_tests.msvc:
+        env_tests.Append(CCFLAGS=["/bigobj"])
```

Comment:
Don't forget to remove this after rebase because it's going to be added in #44808 as needed by other PRs.

I moved this code outside of `env["module_gdnative_enabled"]` there because this applies to any test, not just GDNative.

Reply:
Thanks, rebased and removed.